packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1168,6 +1169,7 @@ abstract class Element implements BuildContext {
 
   Map<Type, InheritedElement> _inheritedWidgets;
   Set<InheritedElement> _dependencies;
+  Set<Type> _unsatisfiedDependencies;
```

Comment:
Why not use _inheritedWidgets with type mapped to null to represent unsatisfied dependencies?